src/config/Config.zig

```diff
@@ -2352,6 +2352,9 @@ keybind: Keybinds = .{},
 /// If `true`, the bold text will use the bright color palette.
 @"bold-is-bright": bool = false,
 
+/// Set color to be used for foreground while rendering bold text.
+@"bold-color": ?Color = null,
```

Comment:
I think we should unify `bold-color` and `bold-is-bright` since they're mutually exclusive. Maybe a custom value here that parses a tagged union.